docs/source/features/features.rst

```diff
@@ -0,0 +1,233 @@
+.. _label_features:
+
+********
+Features
+********
+
+Appleseed supports amongst other features
+
+    * fully programmable shading via `Sony Pictures Imageworks <http://www.imageworks.com/>`_’ `Open Shading Language (OSL) <https://github.com/imageworks/OpenShadingLanguage>`_ 
+    * RGB/spectral/mixed rendering
+    * fast and robust transformation and deformation motion blur
+    * state-of-the-art ray traced subsurface scattering
+    * exhaustive Python and C++ APIs
+    * and many other production-oriented features.
+
+.. _label_features_ecosystem:
+
+Ecosystem
+=========
+
+Along with the core renderer, the team is actively developing high quality integrations
+
+    * appleseed for Autodesk® Maya® → `appleseed-maya <https://github.com/appleseedhq/appleseed-maya>`_
+    * appleseed for Autodesk® 3ds Max® → `appleseed-max <https://github.com/appleseedhq/appleseed-max>`_
+    * appleseed for Blender®  → `blenderseed <https://github.com/appleseedhq/blenderseed>`_ 
+
+appleseed is also the default rendering engine of `Image Engine’s Gaffer <http://www.gafferhq.org/>`_.
+
+.. _label_features_detailed_features_list:
+
+Detailed Features List
+======================
+
+.. _label_features_light_transport:
+
+Light Transport
+---------------
+
+ * Unidirectional Path Tracing [#]_
+ * Stochastic Progressive Photon Mapping [#]_
+ * Light Tracing
+
+.. _label_features_reflection_models:
+
+Reflection Models
+-----------------
+
+ * Lambertian BRDF (purely diffuse)
```

Comment:
I've slightly edited this list on the web site since the time you copied it. Might be worth synchronizing the two.